foundry/pixi/containers/canvasLayers/placeablesLayers/wallsLayer.d.ts

```diff
@@ -0,0 +1,212 @@
+/**
+ * The Walls canvas layer which provides a container for Wall objects within the rendered Scene.
+ * @see {@link Wall}
+ */
+declare class WallsLayer extends PlaceablesLayer<Wall> {
+  constructor();
+
+  /**
+   * A graphics layer used to display chained Wall selection
+   */
+  chain: null;
```

Comment:
This should be `PIXI.Graphics | null` and the `null` should be documented as default value.